tests/assets/layout/default/xml/ltplite.xml

```diff
@@ -0,0 +1,3 @@
+<task name="/kernel/distribution/ltp/lite" role="STANDALONE">
+	<fetch url="https://github.com/RH-FMK/tests-beaker/archive/staging.zip#distribution/ltp/lite"/>
```

Comment:
This is not important for tests, of course, but it might be good to update those URLs.